code/game/dna/genes/vg_powers.dm

```diff
@@ -156,7 +156,7 @@ Obviously, requires DNA2.
 	..()
 	M.update_colour(NOIR_ANIM_TIME)
 	if(M.client) // wow it's almost like non-client mobs can get mutations!
-		M.client.screen += noir_master
+		//M.client.screen += noir_master   Now added by update_colour()
```

Comment:
Might as well delete this line instead of commenting it out.

Reply:
```suggestion

```
I guess